sp800-63b/references.md

```diff
@@ -63,52 +63,52 @@
 
```

Comment:
Why are all the IETF references including the DOI in the reference in addition to the URL? Seems redundant and we aren't doing that for NIST publications, for example.